src/App/Pages/ViewRecipe.js

```diff
@@ -0,0 +1,40 @@
+import React, { useContext } from 'react';
+import { Link } from 'react-router-dom';
+import { jsPDF } from "jspdf";
+import './ViewRecipe.css';
+import { Context } from '../../Hooks/Context';
+
+function ViewRecipe() {
+    
+
+// Convert HTML content to PDF
+const print = (content) => {
+ 
+    const doc = new jsPDF();
```

Comment:
This code is not needed and is duplicated.  The code is not needed since the functionality is not counted in the AC.